openwisp_radius/templates/openwisp-radius/admin/rad_batch_users_add_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'admin/change_form.html' %}
+{% load i18n %}
+{% block field_sets %}
+{% if help_text %}
+<div class="ow-help-text">
+    <p>{{ help_text.text }}
```

Comment:
I removed the image tag from here because it is not looking good with long text.

![image](https://user-images.githubusercontent.com/56037184/161367892-f4a52001-6006-466f-a630-056d8de554fc.png)